.flowconfig

```diff
@@ -1,7 +1,5 @@
 [ignore]
 
-.*/node_modules/fbjs/*
```

Comment:
I mean I can ignore it myself but I think it will cause 1-2hrs of pain for others.